src/components/Register.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+function Register() {
+  const [newEmail, setNewEmail] = useState('');
+  const [newPassword, setNewPassword] = useState('');
+  const [error, setError] = useState(null);
+
+  async function registerForm(e) {
+    e.preventDefault();
+    try {
+      const response = await fetch(`https://localhost:8000/api/v1/auth/register`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          username: newEmail,
```

Comment:
I think username should be something other than email and that there should be an input for it on this form. I believe we will use it to display messages to the user, like a greeting on the navbar after login.